.php_cs.dist.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+$finder = Symfony\Component\Finder\Finder::create()
```

Comment:
Woah, Symfony is a big fat dependency for such a simple algorithm. 

Isn't there a simple glob pattern to configure PhpCsFixer? I use it on another project, and don't need to configure any of this. 

https://github.com/jamesread/Sicroc/blob/master/phpcs.xml